sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -652,6 +653,8 @@ export interface DownloadFromBlobOptions {
   parallelism?: number;
 }
 
+export type GenerateSASUrlOptions = Omit<BlobSASSignatureValues, "blobName" | "containerName">;
```

Comment:
This is an amazing feature! I wonder what the coding experiences would be though. Would Go to Definition come here or go to `BlobSASSignatureValues` directly?  Some time it is easier to understand what's available than what's not available.   Maybe `Pick` is better? /cc @bterlson